doc/sphinx/user/index.md

```diff
@@ -4,6 +4,7 @@
 ---
 maxdepth: 2
 ---
+authors.md
```

Comment:
I would prefer if this is not the first page in the documentation, it seems a bit excessive (also we already link to it from the start page). Can we put it last, after `answers.md`?